apps/examples/touchscreen/touchscreen_main.c

```diff
@@ -168,14 +168,47 @@ static int touchsceen_specific_cmd(int argc, char*argv[])
 	return OK;
 }
 
+static int touchsceen_suspend(void)
+{
+	int fd = open(TOUCH_DEV_PATH, O_RDWR);
+	if (fd < 0) {
+		printf("Fail to open %s, errno:%d\n", TOUCH_DEV_PATH, get_errno());
+		return ERROR;
+	}
+
+	if (ioctl(fd, TSIOC_SUSPEND, NULL) != OK) {
+		printf("Fail to TSIOC_SUSPEND %s, errno:%d\n", TOUCH_DEV_PATH, get_errno());
```

Comment:
When the ioctl is failed, it also returns OK. Is this ok?

Reply:
Updated